src/kernels/raw/finder/localPythonAndRelatedNonPythonKernelSpecFinder.node.ts

```diff
@@ -76,16 +76,11 @@ export class LocalPythonAndRelatedNonPythonKernelSpecFinder extends LocalKernelS
         resource: Resource,
         cancelToken?: CancellationToken
     ): Promise<(LocalKernelSpecConnectionMetadata | PythonKernelConnectionMetadata)[]> {
-        const activeInterpreterPromise = this.extensionChecker.isPythonExtensionInstalled
-            ? this.interpreterService.getActiveInterpreter(resource)
-            : undefined;
         const interpreters = this.extensionChecker.isPythonExtensionInstalled
             ? await this.interpreterService.getInterpreters(resource)
             : [];
-        const activeInterpreter = activeInterpreterPromise ? await activeInterpreterPromise : undefined;
```

Comment:
I noticed this was redundant. getInterpreters already includes the active interpreter.

Reply:
yes, was planning on doing that. python extension added that feature (agreed i filled a bug) recently in the very latest release.
we added it to ensure things work as expected.